ir/src/main/scala/org/scalajs/ir/ScalaJSVersions.scala

```diff
@@ -34,6 +34,11 @@ class VersionChecks private[ir] (
 
   private val (binaryMajor, binaryMinor, binaryPreRelease) = parseBinary(binaryEmitted)
 
+  /** The binary version as advertised in released artifacts. */
+  final val binaryVersion =
```

Comment:
This does not strike me as a very good name. Essentially this is the version of the ecosystem of libraries. Perhaps name it `ecosystemVersion`? Or `artifactsVersion`?

Reply:
I guess `binaryVersion` is sbt jargon (`CrossVersion.binary` vs `CrossVersion.full`). How would you feel about `binaryCross`? (the `Version` suffix is pointless anyways; everything is a version in this class...).

Reply:
Yeah `binaryCross` sounds reasonable.

Reply:
What shall we do about `ScalaJSCrossVersion.currentBinaryVersion`? Remove?

Reply:
Yes. Google and GitHub searches say that no one uses that.